ledger-core/network/nds/msgdelivery/controller.go

```diff
@@ -22,8 +22,12 @@ import (
 
 const minHeadBatchWeight = 1 << 20
 
-func NewController(pt uniproto.ProtocolType, factory nwapi.DeserializationFactory,
-	receiveFn ReceiverFunc, resolverFn ResolverFunc, logger uniserver.MiniLogger,
+func NewController(
```

Comment:
please avoid such code-style changes - it is rare to use this style in the current code base, so it is better to keep it in the original form.

Reply:
@cyraxred Fixed